perllib/FixMyStreet/Script/ArchiveOldEnquiries.pm

```diff
@@ -44,6 +46,63 @@ sub archive {
         printf "Re-run with --commit to actually archive reports.\n\n";
     }
 
+    if ( $opts->{reports} ) {
+        close_list()
+    } elsif ( $opts->{closure_cutoff} ) {
+        close_with_emails();
+    }
+}
+
+sub close_list {
+    my $reports = get_ids_from_csv();
+    my $max_reports = scalar @$reports;
+
+    my $rs  = FixMyStreet::DB->resultset("Problem")->search({
+        id => $reports,
+        state => [ FixMyStreet::DB::Result::Problem->open_states() ],
+    })->to_body( $opts->{body});
+
+    my $no_message = $rs->search({
+        lastupdate => { '<', $opts->{email_cutoff} },
```

Comment:
Yeah, I'm a bit confused by this. So for the normal archive usage, it doesn't do anything for later than email_cutoff, sends an Archive email for earlier than email_cutoff, later than closure_cutoff, and doesn't send anything for earlier than closure_cutoff.

But with the ID list, you're saying send any normal alerts for later than email cutoff, and don't send any normal alerts for earlier than email cutoff. Which is sort of the reverse, iyswim. I think it would make more sense to be using closure_cutoff here as a cutoff between email/no email, and ignore email_cutoff. I think?